docs/UsingOmnisci/index.rst

```diff
@@ -22,5 +22,47 @@ or turn it on in source code:
    cfg.StorageFormat.put('omnisci')
    cfg.IsExperimental.put(True)
 
+Also, only ``Native`` is activated, when you connect it, ``OmniSci`` will connect automatically:
```

Comment:
Probably let's not confuse user much? Just say smth like "to enable omnisci engine launch `export MODIN_STORAGE_FORMAT=omnisci` or use `cfg.StorageFormat.put('omnisci')` in your code"